torchvision/prototype/transforms/functional/_color.py

```diff
@@ -101,7 +101,7 @@ def adjust_contrast(inpt: features.InputTypeJIT, contrast_factor: float) -> feat
 
 
 def adjust_sharpness_image_tensor(image: torch.Tensor, sharpness_factor: float) -> torch.Tensor:
-    num_channels, height, width = get_dimensions_image_tensor(image)
+    num_channels, height, width = image.shape[-3:]
```

Comment:
Aligns the idiom with other parts of the code-base where we directly rely on `shape` to determine the dimensions:
https://github.com/pytorch/vision/blob/70faba91d1c79f689ac6ebd30ad0c4be62690196/torchvision/prototype/transforms/functional/_geometry.py#L539

This make sense as at that point the `image` is a pure tensor and making additional method calls to fetch the sizes is unnecessary.